setup.cfg

```diff
@@ -24,14 +24,14 @@ install_requires =
     nibabel >= 3.0.1
     nipype >=1.3.1
     nitime
-    niworkflows ~= 1.1.6
+    niworkflows @ git+https://github.com/poldracklab/niworkflows.git@edeea81352f3f2b276f11e42e7829e1c5a03770f
```

Comment:
```suggestion
    niworkflows ~= 1.1.7
```

Reply:
Let's do this on release prep and get things merged now.